src/trace/trace.c

```diff
@@ -96,9 +96,22 @@ static inline void mtrace_event(const char *data, uint32_t length)
 }
 #endif /* CONFIG_TRACEM */
 
+/**
+ * \brief Runtime trace filtering
+ * \param lvl log level (LOG_LEVEL_ ERROR, INFO, DEBUG ...)
+ * \param uuid uuid address
+ * \return false when trace is filtered out, otherwise true
+ */
+static inline bool trace_filter_pass(uint32_t lvl,
+				     const struct tr_ctx *ctx)
+{
+	/* LOG_LEVEL_CRITICAL has low value, LOG_LEVEL_VERBOSE high */
+	return lvl <= ctx->level;
```

Comment:
Does it make more sense to to critical as max and verbose debug as the lowest integers. This is what Linux does and it also aligns with IRQ priorities too. i.e. higher number means more importnant.

Reply:
Well, I can change number values, but it will be major debug abi change.

Reply:
oh, dont bother then